tekton/block-tekton-task-runs/block-tekton-task-runs.yaml

```diff
@@ -30,8 +30,10 @@ spec:
     preconditions:
       all:
       - key: "{{ request.operation }}"
-        operator: Equals
-        value: CREATE
+        operator: AnyIn
+        value:
```

Comment:
should we also add `PATCH`?

Reply:
No such thing from a webhook standpoint.